pubspec.lock

```diff
@@ -1060,10 +1060,10 @@ packages:
     dependency: "direct dev"
```

Comment:
Looks like the update to this file is unintended.

Reply:
Yes, it was unintended. It was changed when I ran a `flutter pub get`